compiler/src/dotty/tools/dotc/transform/TypeUtils.scala

```diff
@@ -51,41 +51,44 @@ object TypeUtils {
 
     /** The arity of this tuple type, which can be made up of EmptyTuple, TupleX and `*:` pairs,
      *  or -1 if this is not a tuple type.
-     *
-     *  @param relaxEmptyTuple if true then TypeRef(EmptyTuple$) =:= EmptyTuple.type
      */
-    def tupleArity(relaxEmptyTuple: Boolean = false)(using Context): Int = self match {
```

Comment:
why was this parameter removed? will erasure remain the same for types where `EmptyTuple.type` was widened to `TypeRef(EmptyTuple$)`?

Reply:
ok it seems to work for refined printer I see you made the change, disregard all :)

Reply:
For other that may read this thread: the important thing here is that `tupleElementTypes` handles the `EmptyTuple` type alias but  `tupleArity` does not (see comment on dealiasing)